src/app/app-routing.module.ts

```diff
@@ -1,16 +1,28 @@
 import { NgModule } from '@angular/core';
 import { Routes, RouterModule } from '@angular/router';
+import { AuthGuard } from './modules/private-modules/auth-gaurd/auth.gaurd';
+import { PUBLIC_ROUTES } from './modules/full-layout/layout.routes';
+import { DesignComponent } from './modules/full-layout/design/design.component';
+import { AppComponent } from './app.component';
+import { PrivateGuard } from './modules/community-core';
+import { initializer } from 'src/utils/app-init';
+
+
+
+
 const routes: Routes = [
- 
+
   {
-    path:'',
-    loadChildren: './modules/landing/landing.module#LandingModule'
+    path: 'home',
+    loadChildren: () => import('./modules/landing/landing.module').then(m => m.LandingModule)
   },
+
   {
-    path:'',
-    redirectTo:'',
-    pathMatch:'full'
-  }
+    path: '',
+    // canActivate: [PrivateGuard],
+    loadChildren: () => import('./modules/full-layout/full-layout.module').then(m => m.FullLayoutModule)
```

Comment:
It  should be renamed as a private module, as discussed

Reply:
Renamed